app/views/brexit_checker_mailer/change_notification.text.erb

```diff
@@ -11,14 +11,14 @@
 <% if @action.exception.present? %><%= @action.exception %><% end %>
 
 <% if @action.guidance_url.present? %>
-<%= @action.guidance_prompt || t("brexit_checker_mailer.change_notification.guidance_prompt") %>
```

Comment:
What's the thinking behind doing this in the template instead of [here in the yml](https://github.com/alphagov/finder-frontend/blob/bc6dbbc959a9f0c18401354d0f8614a6650ac34c/config/locales/en/brexit_checker_mailer/change_notification.yml#L4)

Reply:
:thinking: we do seem to do the same thing here:
https://github.com/alphagov/finder-frontend/blob/d881befeb68af24720ec35af92484c9dc089313c/app/views/brexit_checker/_action_list.html.erb#L52

Is there anywhere we're using this without the `:`?

Reply:
Yes e.g. [here](https://github.com/alphagov/finder-frontend/blob/f55dfc4bcb5e29926629e503ebd4c42b720f937e/app/lib/brexit_checker/actions.yaml#L12). It seems to be pretty unnecessary, but unfortunately we don't have full control of the content here. Looking over the file, it always makes sense to add the colon after, but would be impractical to try and add it consistently in `actions.yaml` / the-mighty-spreadsheet.

Reply:
Ace thanks for confirming!